prover/evaluate.py

```diff
@@ -166,7 +166,10 @@ def main() -> None:
     logger.info(
         f"Evaluation done! {num_proved} theorems proved, {num_failed} theorems failed, {num_discarded} non-theorems discarded"
     )
-    logger.info(f"Pass@1: {num_proved / (num_proved + num_failed)}")
+    if num_proved + num_failed == 0 :
+        logger.info(f"Pass@1 : 0")
```

Comment:
I think the `Pass@1` here would be `NaN` instead of 0. Also, we don't need `f` here.

Reply:
Oh, that's more reasonable `NaN` instead of 0. Thanks a lot!

Reply:
I think so, too. I change them!